calendar/src/Components/App.tsx

```diff
@@ -61,19 +55,14 @@ function App() {
 						selectedDate={selectedDate}
 						dateFormatter={dateFormatter}
 						events={events}
-						openModal={openModal}
-						hideModal={hideModal}
+						openModal={() => {}}
+						hideModal={() => {}}
 						saveToLocalStorage={saveToLocalStorage}
 						removeFromLocalStorage={removeFromLocalStorage}
 						timestamp={0}
 					/>
 				</section>
 			</main>
-			<Modal
```

Comment:
I think it was fine to keep `<Modal />` here in one place instead of rendering it with for example a button. Now each time you will need to show the modal, you will have to include into your component and for example for a button, rendering a modal isn't it's responsibility, it's responsibility would be: on button click, call a function to show the modal.